src/mongo/delegates/Views.class.php

```diff
@@ -487,7 +501,7 @@ public function generateView($viewId, $resource = null, $context = null, $queueN
                         _ID_CONTEXT => $doc['_id'][_ID_CONTEXT],
                         _ID_TYPE=>$viewSpec['_id']
                     ],
-                    \_CREATED_TS => \Tripod\Mongo\DateUtil::getMongoDate()
```

Comment:
This was moved to before we fetch the source document to prevent invalidation timing issues.